adapters/consumable/consumable.go

```diff
@@ -18,6 +18,15 @@ type ConsumableAdapter struct {
 	endpoint string
 }
 
+type regsExt struct {
```

Comment:
These structures are already defined as `openrtb_ext.ExtRegs` and `openrtb_ext.ExtUser`. Please see my other comment on helper code you can use or consider using the shared structures.

Reply:
Thanks for the heads up. I did search for existing definitions but I couldn’t find them. Not sure how I missed them, too many occurrences of `us_privacy` and `gdpr` to dig through perhaps :-).

Reply:
Perhaps. :)